src/styles/blocks/header.scss

```diff
@@ -0,0 +1,53 @@
+.header {
+  @include flex-space-beetwen;
+  align-items: center;
+  box-shadow: 0.1px 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    padding: 10px 50px;
+  }
+
+  &__nav-list {
+    display: flex;
+    list-style: none;
+    margin-right: 50px;
+  }
+  
+  &__nav-item {
+    padding: 0 10px;
+  }
+  
+  &__nav-link {
+    @include flex-center;
+    text-align: center;
+    height: 60px;
+    color: black;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+  
+  &__nav-link:hover {
+    color: $button-color;
+  }
+
+}
+
+.is-active {
+  color: $button-color;
+  position: relative;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  border-radius: 8px;
+  bottom: 0;
+  background-color: $button-color;
+  height: 4px;
+  display: block;
+  width: 100%;
+}
+
+.page-content__offset {
+  margin-bottom: 50px;
+}
```

Comment:
Remember - if you create a block (`page-content` here) - styles for it and all its elements should be in a separate file with same name as block.

Reply:
Sure, you're right. It just slipped my mind.